caffe2/python/operator_test/conv_test.py

```diff
@@ -548,8 +565,8 @@ def test_convolution_sync(self, net_type, num_workers, do, engine):
                     weight_init=('ConstantFill', dict(value=w2)),
                     bias_init=('ConstantFill', dict(value=b2)),
                     deterministic=1,
-                    cudnn_state=np.random.randint(0, 3),
-                    use_cudnn=use_cudnn,
+                    gpu_engine_state=np.random.randint(0, 3),
```

Comment:
I think this should be cudnn_state for now as we don't have gpu_engine state defined anywhere else and is passed to the operator directly. Or an other way could be pass miopen_state if has_hip else cudnn_state.

Reply:
Good catch. Changed it back to cudnn_state